R/run_stress_test.R

```diff
@@ -88,6 +123,11 @@ run_stress_test <- function(asset_type,
   cat("-- Exported results to designated output path. \n")
 }
 
+run_example <- function() {
```

Comment:
i wonder if this would not be suited better in utils_helper.R

Reply:
Yes, could be. My reason for defining it here is that it applies to only this function and I would love this hack to not be needed anywhere else.